lyricsgenius/genius.py

```diff
@@ -539,7 +539,7 @@ def search_song(self, title, artist="", get_full_info=True, public_api=False):
             return None
 
         # Download full song info (an API call) unless told not to by user
-        song_info = result.copy()
```

Comment:
Why was there a need to copy the result here?

Reply:
I haven't the faintest clue, fine to delete it like you've done.